src/engine/threaded_engine.h

```diff
@@ -360,6 +380,21 @@ class ThreadedEngine : public Engine {
     }
   }
 
+  /*! \brief Return the number of OMP threads that should be used per worker
+   * \return Number of OMP threads that should be used per worker
+   */
+  int GetNumOMPThreadsPerWorker() const override {
```

Comment:
we follow the google c++ style guide.

Accessor and mutator should be named as num_omp_threads_per_worker() and set_ num_omp_threads_per_worker()

Reply:
ok.  So in a class with many CamelCase names, anything with get and set is lowercase?